chat/attachment.go

```diff
@@ -16,7 +16,7 @@ type Attachment struct {
 	AuthorName string `json:"author_name,omitempty"`
 
 	// AuthorLink is a valid URL that will hyperlink the author_name text mentioned above.
-	AuthorLink string `json:"author_name,omitempty"`
+	AuthorLink string `json:"author_link,omitempty"`
```

Comment:
This is technically a separate fix and should be in a separate commit, as it's not related to the markdown support change.